wp-content/themes/core/assets/js/admin/stacking-order.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * @module stacking-order
```

Comment:
This will end up in an editor (rather than admin) enqueue with your other PR, correct?